dart-impl/mpi/src/dart_mem.c

```diff
@@ -166,7 +171,7 @@ dart_buddy_alloc(struct dart_buddy * self, size_t s) {
 		}
 		for (;;) {
 			level--;
-			length *= 2;
+//    length *= 2;
```

Comment:
It's a dead increment: https://felixmoessbauer.de/temp/build-report/report-fa474c.html#EndPath